app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -8,7 +8,7 @@
   <%= @presenter.phone_number_message %>
 </p>
 
-<%= validated_form_for(:login_otp, method: :post) do %>
+<%= validated_form_for('', method: :post) do |f| %>
```

Comment:
This feels _weird_ to me, but seems to work as a workaround to avoid SimpleForm inputs from prefixing the "model" name. Without this, the field below becomes `name="login_otp[remember_device]"`, meaning we have to change how we process the input on the server. My impression is that we just made up symbols to use here, since the form isn't actually associated with a model, so the name (or absence of a name) doesn't really matter.

Reply:
also every time we change field names like this, the load test scripts break... so IMO it would be OK to leave it with the weird verbose old name

Reply:
Just to clarify, it would have changed from `name="remember_device"` to `name="login_otp[remember_device]"` if we kept `:login_otp`, but by setting the first parameter to the empty string, it keeps the same old name. To your point, this helps avoid a lot of changes/potential for breakage like other times I've had to deal with introducing SimpleForm helpers in place of Rails field tag helpers.

Reply:
ah sorry I got it backwards, but yes, anything to minimize noise